lib/SPIRV/LLVMIntrinsicEmulation/small_bitreverse.c

```diff
@@ -0,0 +1,29 @@
+// clang-format off
```

Comment:
One thing that will be beneficial to developers will be to add a short doc titled 'LLVM intrinsic emulation'. It need not be part of this PR, but I will suggest to add it sooner when things are fresh in memory. Thanks

Reply:
Totally agree